src/resources.json

```diff
@@ -126,6 +126,14 @@
                   "method": "GET",
                   "path": "/{id}/feed-items/all"
                 },
+                "listAllAvailablePromoCodes" : {
```

Comment:
Maybe we can remove the word 'All' here since it might be confusing for paginated purposes, and it doesn't give the dev more info on what the endpoint does. `listAvailablePromoCodes` should be fine, same for unavailable codes  :)

Reply:
aye good call